generated/elasticsearch/composable/component/email.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
I missed including this new file in https://github.com/elastic/ecs/pull/1688.

I can address this as a separate PR outside this issue.

Reply:
Resolved in https://github.com/elastic/ecs/pull/1705